__tests__/authRoutes.test.js

```diff
@@ -4,13 +4,15 @@ import chaiHttp from 'chai-http';
 import app from '..';
 import models from '../database/models';
 import mockUsers from './mockData/mockUsers';
+import mockUsersToVerify from './mockData/mockUsersToVerify';
```

Comment:
Destructure the values coming from `mockData`

Reply:
I can't use the values coming from mockUsers because their emails would have been signed up by the time I need to run the signup before hooks for my test. That is why I created my own mockUsersToVerify